xrpl/asyncio/transaction/main.py

```diff
@@ -142,6 +147,72 @@ async def submit_transaction(
     raise XRPLRequestFailureException(response.result)
 
 
+async def safe_sign_transaction_with_multisign(
+    transaction: Transaction,
+    wallet: Wallet,
+) -> str:
+    """
+    Signs a transaction with to be used for multisigning.
+
+    Args:
+        transaction: the transaction to be signed.
+        wallet: the wallet with which to sign the transaction.
+
+    Returns:
+        The signed transaction blob.
+    """
+    signature = sign(
```

Comment:
a sidenote: we should add support for `sign` to accept hex or bytes, and convert between them